language-design.md

```diff
@@ -74,7 +104,9 @@ the commenters?
 
 **Response**
 
+The comments took on a variety of tones. Some programmers want to consider themselves as [special snowflakes](http://i.imgur.com/AfnKFeA.jpg) with superior intellects. For example, one guest user claims that the reason some people can’t understand the syntax is because, “Most likely because their brains do not process information like a real programmer does.” [Anonymous Guest](https://www.fastcodesign.com/1665735/why-arent-computer-programming-languages-designed-better#comment-2dfe2900-7795-11e2-9384-9794fc737ae8). However, other people had more reasonable approaches.
```

Comment:
I think it's interesting that you chose to mention these commenters first. In our response, we tried to focus on the average tone of the comments, as you did below. By including the viewpoints of these commenters, it shows more of the variety in the comments and backs it up with specific examples.